### PR TITLE
Feature/zh cn resource quota v1

### PR DESCRIPTION
**Pull request by donghui12**
Tranlate ontent/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md
https://github.com/kubernetes/website/issues/34733

### CODE REVIEW

**Review by windsonsea — CHANGES_REQUESTED** [review]
kick a beginning nits.

You can modify some layout alignment issues by preview: 
https://deploy-preview-34933--kubernetes-io-main-staging.netlify.app/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1/

**Comment by tengqm** [comment]
pls squash your commits to 'main'. This is a new file and there should be no commit related to other files.
Please also always keep your PR at single commit.

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md [hunk 1]
```diff
@@ -0,0 +1,817 @@
...
+ResourceQuotaSpec 定义为 Quota 强制执行所需的硬限制。
+
+<hr>
+
+- **hard** (map[string]<a href="{{< ref "../common-definitions/quantity#Quantity" >}}">Quantity</a>)
+
+  <!-- 
+  hard is the set of desired hard limits for each named resource. More info: https://kubernetes.io/docs/concepts/policy/resource-quotas/
+  -->
+  hard 是每个指定资源所需的硬限制集。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md:77]:
```suggestion
  hard 是每种指定资源所需的硬性限制集合。
```

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md [hunk 2]
```diff
@@ -0,0 +1,817 @@
...
+  更多信息： https://kubernetes.io/docs/concepts/policy/resource-quotas/
+
+- **scopeSelector** (ScopeSelector)
+
+  <!-- 
+  scopeSelector is also a collection of filters like scopes that must match each object tracked by a quota but expressed using ScopeSelectorOperator in combination with possible values. For a resource to match, both scopes AND scopeSelector (if specified in spec), must be matched. 
+  -->
+  scopeSelector 也就是过滤器的集合，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md:85]:
```suggestion
  scopeSelector 也是一组过滤器的集合，和 scopes 类似，
```

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md [hunk 3]
```diff
@@ -0,0 +1,817 @@
...
+  例如必须匹配由配额跟踪但使用 ScopeSelectorOperator 结合可能值表示的每个对象的范围。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md:86]:
```suggestion
  必须匹配配额所跟踪的每个对象，但使用 ScopeSelectorOperator 结合可能的值来表示。
```

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md [hunk 4]
```diff
@@ -0,0 +1,817 @@
...
+  对于要匹配的资源，必须匹配范围和范围选择器（如果在规范中指定）。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md:87]:
```suggestion
  对于要匹配的资源，必须同时匹配 scopes 和 scopeSelector（如果在 spec 中设置了的话）。
```

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md [hunk 5]
```diff
@@ -0,0 +1,817 @@
...
+
+  <a name="ScopeSelector"></a>
+  <!-- 
+  *A scope selector represents the AND of the selectors represented by the scoped-resource selector requirements.* 
+  -->
+  scope selector 表示 scoped-resource selector 所代表的**与**选择器。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md:93]:
```suggestion
  scope 选择算符表示的是由限定范围的资源选择算符进行 **逻辑与** 计算得出的结果。
```

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md [hunk 6]
```diff
@@ -0,0 +1,817 @@
...
+
+  - **scopeSelector.matchExpressions** ([]ScopedResourceSelectorRequirement)
+
+    <!-- 
+    A list of scope selector requirements by scope of the resources. 
+    -->
+    按资源范围划分的 scope-selector-requirement 列表。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md:100]:
```suggestion
    按资源范围划分的范围选择算符需求列表。
```

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md [hunk 7]
```diff
@@ -0,0 +1,817 @@
...
+
+    <a name="ScopedResourceSelectorRequirement"></a>
+    <!-- 
+    *A scoped-resource selector requirement is a selector that contains values, a scope name, and an operator that relates the scope name and values.* 
+    -->
+    scoped-resource-selector-requirement 是包含值、范围名称和关联两者的选择器。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md:106]:
```suggestion
    限定范围的资源选择算符需求是一种选择算符，包含值、范围名称和将二者关联起来的运算符。
```

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md [hunk 8]
```diff
@@ -0,0 +1,817 @@
...
+
+    - **scopeSelector.matchExpressions.operator** (string)，必需
+
+      <!-- 
+      Represents a scope's relationship to a set of values. Valid operators are In, NotIn, Exists, DoesNotExist. 
+      -->
+      表示范围与值的关系。有效的运算符为 In、NotIn、Exists、DoesNotExist。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md:113]:
```suggestion
      表示范围与一组值之间的关系。有效的运算符为 In、NotIn、Exists、DoesNotExist。
```

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md [hunk 9]
```diff
@@ -0,0 +1,817 @@
...
+
+    - **scopeSelector.matchExpressions.scopeName** (string)，必需
+
+      <!-- 
+      The name of the scope that the selector applies to. 
+      -->
+      选择器应用于作用域的名称。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md:120]:
```suggestion
      选择器所适用的范围的名称。
```

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md [hunk 10]
```diff
@@ -0,0 +1,817 @@
...
+
+    - **scopeSelector.matchExpressions.values** ([]string)
+
+      <!-- 
+      An array of string values. If the operator is In or NotIn, the values array must be non-empty. If the operator is Exists or DoesNotExist, the values array must be empty. This array is replaced during a strategic merge patch. 
+      -->
+      字符串值数组。
+      如果操作符是 In 或 NotIn，values 数组必须是非空的。
+      如果操作符是 Exists 或 DoesNotExist，values 数组必须为空。
+      该数组将在战略合并补丁期间替换。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md:130]:
```suggestion
      该数组将在策略性合并补丁操作期间被替换。
```

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md [hunk 11]
```diff
@@ -0,0 +1,817 @@
...
+
+- **scopes** ([]string)
+
+  <!-- 
+  A collection of filters that must match each object tracked by a quota. If not specified, the quota matches all objects. 
+  -->
+  一个匹配每个被配额跟踪的对象的筛选器集合。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md:137]:
```suggestion
  一个匹配被配额跟踪的所有对象的过滤器集合。
```

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md [hunk 12]
```diff
@@ -0,0 +1,817 @@
...
+  如果没有指定，则默认匹配所有对象。
+
+## ResourceQuotaStatus {#ResourceQuotaStatus}
+
+<!-- 
+ResourceQuotaStatus defines the enforced hard limits and observed use. 
+-->
+ResourceQuotaStatus 定义了 hard limit 和 observed use。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md:145]:
```suggestion
ResourceQuotaStatus 定义硬性限制和观测到的用量。
```

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md [hunk 13]
```diff
@@ -0,0 +1,817 @@
...
+
+<hr>
+
+- **hard** (map[string]<a href="{{< ref "../common-definitions/quantity#Quantity" >}}">Quantity</a>)
+
+  <!-- 
+  Hard is the set of enforced hard limits for each named resource. More info: https://kubernetes.io/docs/concepts/policy/resource-quotas/ 
+  -->
+  hard 是每个命名资源的强制硬限制集。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md:154]:
```suggestion
  hard 是每种指定资源所强制实施的硬性限制集合。
```

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md [hunk 14]
```diff
@@ -0,0 +1,817 @@
...
+  更多信息： https://kubernetes.io/docs/concepts/policy/resource-quotas/
+
+- **used** (map[string]<a href="{{< ref "../common-definitions/quantity#Quantity" >}}">Quantity</a>)
+
+  <!-- 
+  Used is the current observed total usage of the resource in the namespace. 
+  -->
+  used 是当前监控的命名空间中资源的总使用量。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md:162]:
```suggestion
  used 是当前命名空间中所观察到的资源总用量。
```

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md [hunk 15]
```diff
@@ -0,0 +1,817 @@
...
+#### HTTP 请求
+
+GET /api/v1/namespaces/{namespace}/resourcequotas/{name}
+
+<!--
+#### Parameters
+-->
+#### 参数
+
+- **name** （*路径参数*）: string, 必需
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md:217]:
```suggestion
- **name** （**路径参数**）: string, 必需
```

下同。

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md [hunk 16]
```diff
@@ -0,0 +1,817 @@
...
+
+  ResourceQuota 的名称
+
+- **namespace** （*路径参数*）: string, 必需
+
+  <a href="{{< ref "../common-parameters/common-parameters#namespace" >}}">namespace</a>
+
+- **pretty** （*查询参数*）: string
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md:225]:
```suggestion
- **pretty** （**查询参数**）: string
```

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md [hunk 17]
```diff
@@ -0,0 +1,816 @@
...
+  *A scope selector represents the AND of the selectors represented by the scoped-resource selector requirements.* 
+  -->
+  scope 选择算符表示的是由限定范围的资源选择算符进行 **逻辑与** 计算得出的结果。
+
+  - **scopeSelector.matchExpressions** ([]ScopedResourceSelectorRequirement)
+
+    <!-- 
+    A list of scope selector requirements by scope of the resources. 
+    -->
+    按资源范围划分的范围选择算符需求列表。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md:99]: 99 前面加空行

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md [hunk 18]
```diff
@@ -0,0 +1,816 @@
...
+    *A scoped-resource selector requirement is a selector that contains values, a scope name, and an operator that relates the scope name and values.* 
+    -->
+    限定范围的资源选择算符需求是一种选择算符，包含值、范围名称和将二者关联起来的运算符。
+
+    - **scopeSelector.matchExpressions.operator** (string)，必需
+
+      <!-- 
+      Represents a scope's relationship to a set of values. Valid operators are In, NotIn, Exists, DoesNotExist. 
+      -->
+      表示范围与一组值之间的关系。有效的运算符为 In、NotIn、Exists、DoesNotExist。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md:112]: 112 前面加空行

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md [hunk 19]
```diff
@@ -0,0 +1,816 @@
...
+
+    - **scopeSelector.matchExpressions.scopeName** (string)，必需
+
+      <!-- 
+      The name of the scope that the selector applies to. 
+      -->
+      选择器所适用的范围的名称。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md:119]: 119 前面加空行

File: content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md [hunk 20]
```diff
@@ -0,0 +1,816 @@
...
+
+    - **scopeSelector.matchExpressions.values** ([]string)
+
+      <!-- 
+      An array of string values. If the operator is In or NotIn, the values array must be non-empty. If the operator is Exists or DoesNotExist, the values array must be empty. This array is replaced during a strategic merge patch. 
+      -->
+      字符串值数组。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/policy-resources/resource-quota-v1.md:126]: 126 前面加空行

**Comment by Sea-n** [comment]
Hey @donghui12,

It's been a while since last update, would you like to address issues in the previous comments so that we can move forward?

**Comment by donghui12** [comment]
I got label lgtm ，think this pr is comleted ，I will fix this pr again

**Pull request merged**